### DISCUSSION TITLE
Name the presets

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#590] [General]
[WAY TO TOP 1](https://static.sonofind.com/audio/aac-bPnyq5/SPOTS/SPOT0011/SPOT0011782.aac) by Dieter Reith ([Source](https://sonoton.com/en/track/SPOT001177/))

**Comment by Banana71** [comment]
404 not found
the link doesn't work for me

**Reply by Piattica** [reply]
For me too

**Comment by Banana71** [comment]
I had better luck with a different browser. I was now able to play the track using the (Source) link.

**Reply by Banana71** [reply]
Bass: ROM1A Patch 15 "Bass 1"
Brass: ROM1A Patch 1 "Brass 1" 
Clav: I suspect a layer of "Clav 2" ROM1B and an electric piano 

Hard to say, it was all shot from the hip
